opencog/nlp/relex2logic/rules/tense.scm

```diff
@@ -0,0 +1,64 @@
+;Both link grammar and relex output are selected because of the pattern 
+;Example : "He gave me the book."
+;(DefinedLinguisticConceptNode "past") and (DefinedLinguisticConceptNode ".v-d")
+;I just want to avoid the Link Grammar output. There might be another way to do this
+(define (check-tense tense)
+    (if (string-contains (cog-name tense) ".")
```

Comment:
You could do something like this, maybe:
```
(NotLink (EqualLink (VariableNode "$tense") (DefinedLinguisticConceptNode ".v-d")))
```

Note, however, relex is broken and should be fixed: it is generating
```
; tense (run, past)
(InheritanceLink (stv 1.0 1.0)
   (WordInstanceNode "ran@6bba3d6e-3b62-40d2-a24b-634352abc6c3")
   (DefinedLinguisticConceptNode "past")
)
; subscript-TAG (run, .v-d)
(InheritanceLink (stv 1.0 1.0)
   (WordInstanceNode "ran@6bba3d6e-3b62-40d2-a24b-634352abc6c3")
   (DefinedLinguisticConceptNode ".v-d")
)
```
and that is just wrong; one of these should be TenseLink, and the other should be SubscriptLink.  The inheritanceLink is useless, because it gloms two completely different categories together into one.